blog/2024-03-07-cloud-storage/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+slug: cloud-storage
+title: |
+  Cloud storage: gain flexibility and performance for data access for your project in Renku
+authors: [elisabet]
+tags:
+  - Cloud Storage
+  - Data
+  - RenkuLab
+  - Renku CLI
+image: ./image_1.png
+---
+
+![Cloud storage access from Renku](./image_1.png)
+
+Working with your data in Renku is faster and simpler: you can write and read your data from your external storage services, such as S3, Azure Blob or WebDAV among others.
+
+RenkuLab interactive sessions can access data hosted in a cloud storage system, and the storage is simply mounted as another folder in your session. You can concentrate on developing your code to analyse your the data and share the results with your colleagues or even with the broader community.
+
```

Comment:
Could we please add a screenshot of adding cloud storage here? For example, the same one as in the Discourse announcement post? https://renku.discourse.group/t/renkulab-update-universal-cloud-storage-and-managing-sessions/998

Reply:
Added a gif with cloud storage setting